### PR TITLE
Feature/updatesformetamodule

### PR DESCRIPTION
**Pull request by CodeCyclone**
* Scripts for release pipeline
* Appveyor fixes

### CODE REVIEW

File: scripts/DownloadNupkg.ps1 [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        $nugetArgs += '-Prerelease'
+    }
+
+    Write-Verbose "Running: & $nugetExe $($nugetArgs -join ' ')"
+    & $nugetExe $nugetArgs
+    if($LASTEXITCODE -ne 0) {
+        throw "Nuget.exe failed with exit code: $LASTEXITCODE"
+    }
+
+    # $nugetPackageDirectory = Get-ChildItem -Path $OutputDirectory | Where-Object { $_.Name -match "$([regex]::Escape($packageName)).\d+.*" }
```
Comment (nandemoii) [inline scripts/DownloadNupkg.ps1]: Commented out code.

File: scripts/GenerateFileCatalog.ps1 [hunk 2]
```diff
@@ -40,7 +40,9 @@ if(!$nuspecFiles) {
 }
 
 foreach($nuspecFile in $nuspecFiles) {
-    $moduleName = $nuspecFile.BaseName
+    # $moduleName = $nuspecFile.BaseName <- makes it be lowercase
```
Comment (nandemoii) [inline scripts/GenerateFileCatalog.ps1]: Is this sometimes necessary? It looks like you're using FullName now.

File: scripts/Test.ps1 [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    # Indicates to upload test results to AppVeyor.
+    [switch] $UploadResultsToAppVeyor
+)
+
+Write-Output "Running $($MyInvocation.MyCommand.Name)"
+
+$Path = (Resolve-Path -Path $Path -ErrorAction Stop).ProviderPath
+
+# https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-test?tabs=netcore2x
+
```
Comment (nandemoii) [inline scripts/Test.ps1]: Remove this whitespace.

File: scripts/GenerateFileCatalog.ps1 (hunk 2 above)
Comment (CodeCyclone) [reply scripts/GenerateFileCatalog.ps1]: I will keep it there as a reminder to not do it in this scenario

File: scripts/DownloadNupkg.ps1 (hunk 1 above)
Comment (CodeCyclone) [reply scripts/DownloadNupkg.ps1]: I will remove

File: scripts/Test.ps1 (hunk 3 above)
Comment (CodeCyclone) [reply scripts/Test.ps1]: Will remove

**Pull request merged**